Packs/GitHub/Integrations/GitHub/GitHub.py

```diff
@@ -17,7 +17,7 @@
 USER = demisto.params().get('user')
 TOKEN = demisto.params().get('token', '')
 PRIVATE_KEY = demisto.params().get('credentials', {})
-PRIVATE_KEY = PRIVATE_KEY.get('credentials', {}).get('sshkey', '') if PRIVATE_KEY else PRIVATE_KEY
+PRIVATE_KEY = PRIVATE_KEY.get('sshkey', '') if PRIVATE_KEY else PRIVATE_KEY
```

Comment:
@esharf Told me to change it (he wrote it), I think that the reason is because we already assigned PRIVATE_KEY to be demisto.params().get('credentials', {}) it's looks like a duplication